frontend/src/metabase/lib/colors/palette.ts

```diff
@@ -140,20 +140,3 @@ export const getFocusColor = (
   colorName: string,
   palette: ColorPalette = colors,
 ) => lighten(color(colorName, palette), 0.465);
-
-// We intentionally want to return white text color more frequently
```

Comment:
moved this whole block to `static-viz/lib/colors.ts`

Reply:
I think this logic is generic and does not belong to `static-viz` only. It is used in the dynamic pie chart and potentially can be used in other places